sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -781,7 +781,7 @@ object RapidsConf {
   val ENABLE_ORC_WRITE = conf("spark.rapids.sql.format.orc.write.enabled")
     .doc("When set to false disables orc output acceleration")
```

Comment:
This documentation should warn about the known ORC issue and link to the ORC JIRA for details, otherwise users may wonder why this is disabled by default, shrug their shoulders, and simply enable it always in their pipelines.

Reply:
Done